ci/Dockerfile

```diff
@@ -0,0 +1,33 @@
+
+FROM ubuntu:20.04 AS risingwave-build-env
+
+ENV LANG en_US.utf8
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y install make build-essential cmake protobuf-compiler curl openssl libssl-dev pkg-config bash openjdk-11-jdk wget unzip git tmux
+
+SHELL ["/bin/bash", "-c"]
+
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --no-modify-path --default-toolchain none -y
+
+RUN mkdir -p /risingwave
+
+WORKDIR /risingwave
+
+ENV PATH /root/.cargo/bin/:$PATH
+
+# We need to add the `rustfmt` dependency, otherwise `risingwave_pb` will not compile
+RUN rustup self update \
```

Comment:
Please add to CONTRIBUTING.md to remind users to change this default toolchain if they get this updated.

Reply:
How about we reuse the toolchain definition in source repo?

Reply:
If you want to redefine the toolchain of the build environment, you need to rebuild the image and upload it to aws ecr, or you can separate some tools in the workflow.